src/plugins/recordTypes/organization/advancedSearch.js

```diff
@@ -0,0 +1,47 @@
+export default (pluginContext) => {
+  const {
+    OP_EQ,
+    OP_CONTAIN,
+    OP_RANGE,
+  } = pluginContext.searchOperators;
+
+  const {
+    defaultAdvancedSearchBooleanOp,
+    extensions,
+  } = pluginContext.config;
+
+  return {
+    op: defaultAdvancedSearchBooleanOp,
+    value: [
+      {
+        op: OP_CONTAIN,
+        path: 'ns2:organizations_common/orgTermGroupList/orgTermGroup/termDisplayName',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:organizations_common/orgTermGroupList/orgTermGroup/termStatus',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:organizations_cinefiles/foundingCity',
+      },
+      {
+        op: OP_CONTAIN,
+        path: 'ns2:organizations_cinefiles/foundingState',
+      },
+      {
+        op: OP_CONTAIN,
+        path: 'ns2:organizations_common/foundingPlace',
```

Comment:
Change to OP_EQ. Since foundingPlace is now a dropdown, "contains" search isn't needed, because the user can only select exact values anyway.